content/en/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -170,6 +170,28 @@ to access a replica via its ephemeral public IP, you must skip TLS verification.
 To allow etcd clustering, ports needed to communicate between etcd instances will be opened (for inside cluster communication).
 To make such deployment secure, communication between etcd instances is authorized using SSL.
 
+### API server identity
+
+{{< feature-state state="alpha"  for_k8s_version="v1.20" >}}
+
+The API Server Identity feature is controlled by a feature gate
+and is not enabled by default.  See
+[Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/)
+for a general explanation of feature gates and how to enable and disable them.  The
+name of the feature gate is "APIServerIdentity".  You can enable API Server Identity
+by adding the following command-line flag to your `kube-apiserver` invocation:
+
+```shell
+kube-apiserver \
+--feature-gates=APIServerIdentity=true \
+ # …and other flags as usual
+```
+
+Each API server will be assigned with an unique ID on bootstrap. The list of
```

Comment:
```suggestion
During bootstrap, NAME_OF_THING assigns a unique ID to each API server. Each kube-apiserver
manages a [Lease](/docs/reference/generated/kubernetes-api/{{< param "version" >}}//#lease-v1-coordination-k8s-io) 
in the _kube-apiserver-lease_ {{< glossary_tooltip text="namespaces" term_id="namespace">}}.
The Lease contains the unique ID for the kube-apiserver. If another kube-apiserver tries to join
same control plane, the additional API server [picks a unique ID that is not used within the cluster?]

Enabling this feature [provides benefits?].

Each API server also exposes its own ID as a label dimension in its metrics: `TBD`.
```

Reply:
Updated the mechanism and the benefit. 

The metrics didn't made it into alpha. Dropped the sentence.